documentation/source/user_section/tutorials/lesion-analysis/lesion-statistics/lesion-metrics.rst

```diff
@@ -0,0 +1,60 @@
+Compute lesion morphometric measures using ``sct_analyze_lesion``
+#################################################################
+
+The ``sct_analyze_lesion`` function computes various morphometric measures on segmented lesions and spinal cord.
+In case of multiple lesions, the function assigns an ID value to each lesion (1, 2, 3, etc.) and then outputs morphometric measures for each lesion to an XLS file.
+
+The following morphometric measures are computed:
+
+* ``volume [mm^3]`` : volume of the lesion
+* ``length [mm]`` : length of the lesion along the superior-inferior (SI) axis
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/intramedullary-lesion-length.png
+  :align: center
+  :figwidth: 60%
+
+* ``max_equivalent_diameter [mm]`` : maximum diameter of the lesion, when approximating the lesion as a circle in the axial plane
+* ``max_axial_damage_ratio []`` : maximum axial damage ratio defined as the ratio of the lesion area divided by the spinal cord area. The ratio is computed in the axial plane for each slice and and the maximum ratio is reported.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lesion-analysis/axial-damage-ratio.png
+  :align: center
+  :figwidth: 60%
+
+TODO: Ask Andrew Smith for permission to use the figure
```

Comment:
to address

Reply:
confirmed by email -- we have the permission from Andrew Smith to use the figure
TODO removed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4586/commits/86285acaea993f9798c0861b1c591553b89f8f9f